### ISSUE TITLE
Button style not applied to formly type button with 17.0.13 version

### ISSUE THREAD

**Issue by surendranekkanti** [GSA/sam-design-system#1570]
**Components Involved**
Formly Type "button" 

**Expected Behavior**
Style need to be applied based on the btntype property.

**Actual Behavior**
Button displays with no styling. See screenshots for reference

**Your Environment**
Angular Version: 17.3.9
 "@gsa-sam/components": "17.0.13",
"@gsa-sam/sam-formly": "17.0.13"

**Ticket Number**
Please provide your JIRA ticket number that is dependent on this fix. If there is none, please create
one to validate the issue once the fix has been released. DO NOT provide the url, please ONLY
provide the ticket number.
IAEMOD-36747

**Additional context**
Screenshots / recordings, minimal reproduction on stackblitz, steps to reproduce, code snippets, etc.
![formly-button-comp](https://github.com/user-attachments/assets/a0ca88be-a5f7-43bb-866c-25d9a8ea3bf0)
![formly-button](https://github.com/user-attachments/assets/ebef8d38-be7d-478a-95d9-ff44073c2a46)

**Comment by cwolf10** [comment]
@surendranekkanti can you provide me the section of the formly config where you are specifying that button?

**Comment by cwolf10** [comment]
@surendranekkanti I think I missed the first part of you talking about this issue, are you expecting that the button look like your first screenshot or the second. Based on the version history, I'm guessing you are expecting the first, but wanted to confirm.

**Comment by surendranekkanti** [comment]
@cwolf10 Yes the button should look the one in the first screenshot. Basically this is the formly configuration and we have a custom class(search-zip) that would applied from the btntype property but with the version upgrade I do not see that being applied.

![Uploading image.png…]()
